mx.py

```diff
@@ -13841,6 +13842,7 @@ def warn(msg, context=None):
     'ideclean': [ideclean, ''],
     'ideinit': [ideinit, ''],
     'intellijinit': [intellijinit, ''],
+    'jackpot': [mx_jackpot.jackpot, ''],
```

Comment:
You should also bump the minor version number in the global `version` field near the bottom of `mx.py`.